File: Application Mobile/lib/Models/AdminReports/admin_report_create_request.dart
class AdminReportCreateRequest {
  String name;
  String url;
  int reportTypeId;

  AdminReportCreateRequest({
    this.name = '',
    this.url = '',
    this.reportTypeId = 0,
  });


  Map<String, dynamic> toJson() {
    return {
      'name': name,
      'url': url,
      'report_type_id': reportTypeId
    };
  }
}

File: Services/enerser.analytics/Models/Report/admin_report_create_request.cs
namespace enerser.analytics.Models.Report
{
    public class AdminReportCreateRequest
    {
        public string name { get; set; } = string.Empty;
        public string url { get; set; } = string.Empty;
        public int report_type_id { get; set; } = new int();
    }
}
